setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+setup(
+    name='potato-annotation',
+    version='1.2.0.2',
+    packages=find_packages(),
+    entry_points={
+        'console_scripts': [
+            'potato = potato.cli:potato',
+        ],
+    },
+    url="https://github.com/davidjurgens/potato",
+    author="Jiaxin Pei",
```

Comment:
Can you create a general email and contact point for Potato so that everyone who maintains it can respond?